#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("binary_threshold",	int_t,		0, "Binary Threshold", 192,  0, 255)
gen.add("ht_min_points",	int_t,		0, "HT Min Points", 30,  0, 1000)
gen.add("roi_y",		int_t,		0, "ROI Y (cm)", 70,  0, 1000)
gen.add("road_distance_cm",	double_t,	0, "Road distance (cm)", 15,  0, 1000)
gen.add("lad_cm",		double_t,	0, "Look ahead distance (cm)", 60,  0, 1000)
gen.add("speed_m_s",		double_t,	0, "Speed m/s", 0,  0, 1)

exit(gen.generate("wild_thumper", "path_following", "PathFollowing"))
